smartling-api-commons/src/main/java/com/smartling/api/v2/client/exception/client/AuthenticationErrorException.java

```diff
@@ -0,0 +1,13 @@
+package com.smartling.api.v2.client.exception.client;
+
+import com.smartling.api.v2.response.ErrorResponse;
+
+import javax.ws.rs.core.Response;
+
+public class AuthenticationErrorException extends ClientApiException
```

Comment:
IMO, we should have JavaDocs on these exceptions. I won't add a comment to all of them :)